server/src/main/java/org/apache/iotdb/db/metadata/logfile/MLogWriter.java

```diff
@@ -437,7 +448,7 @@ public static PhysicalPlan convertFromString(String str) {
                 CompressionType.values()[Integer.parseInt(words[5])]));
       case "1":
         return new StorageGroupMNodePlan(
-            words[1], Long.parseLong(words[2]), Integer.parseInt(words[3]));
+            words[1], Long.parseLong(words[2]), Integer.parseInt(words[3]), 0, 0);
       case "0":
```

Comment:
The versions seem not to be persisted.

Reply:
This function is only used for upgrade, in the previous version(<0.13), the node has no versions. so there I use zero.